### ISSUE TITLE
Tools-Docker Make error.

### ISSUE THREAD

**Issue by andycao125** [hyperledger/fabric#3003]
When Tools-Docker compiles 2.3, it will appear as follows:
Make: Go: Operation Not Permitted
Make: Go: Operation Not Permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: Go: Operation Not Permitted
Make: Go: Operation Not Permitted
Make: Go: Operation Not Permitted
Make: Go: Operation Not Permitted
Make: git: Operation Not permitted
Make: / bin / sh: Operation Not Permitted
Make: *** [makefile: 209: build / bin / configtxgen] error 127
Removing intermediate Container D57FC2BAC95C
THE Command '/ Bin / Sh -c Make Configtxgen Configtxlator Cryptogen Peer Discover Osnadmin IdemixGen Go_Tags = $ {Go_Tags}' Returned A Non-Zero Code: 2
Compiled by updating the Alpine_Ver version corresponding to the Makefile

**Comment by berserkr** [comment]
This happened on 2.3... it went away on 2.4 beta... but now I am getting:
 ```The command line help docs are out of date and need to be regenerated
make: *** [Makefile:123: check-help-docs] Error 2```

**Comment by denyeart** [comment]
Please provide detailed reproduction steps and your assessment of what needs to change in Makefile.

**Comment by denyeart** [comment]
@andycao125 @berserkr Is this problem still occurring? Any assessment?

**Comment by berserkr** [comment]
@denyeart I abandoned the branch I was on. I ended up trying multiple branches until I was able to build everything. At the time the other thing that I recalled affected things was the version of golang. The docs and the requirements from the branch had a mismatch in version as I recall. Things working for me are: v2.3.0 with go version go1.16.7 linux/amd64. Go version in the docks for that branch: Go version 1.14.x. Cheers.
